src/main/java/com/redhat/devtools/intellij/tektoncd/actions/RefreshAction.java

```diff
@@ -30,6 +30,9 @@
 
 import javax.swing.tree.TreePath;
 
+import static com.redhat.devtools.intellij.tektoncd.telemetry.TelemetryService.*;
```

Comment:
Starred imports are prohibited

Reply:
weird, even though I have configured IJ to only introduce those once I have 999 imports, it keeps happening.